src/views/LocationEditors/LocationFromAddress.vue

```diff
@@ -11,12 +18,128 @@
       location: null,
```

Comment:
The location is recovered from the local storage on future page loads. This means it will be defined automatically by the parent (Report) component after this component loads. At this time, you need to tell the parent component if the location is valid by emitting 
`this.$emit('update:valid', ...);`
Otherwise the user will have to change the value to make the valid event update.
I can't find this in the code, are you sure it's handled ?